pkg/release/release.go

```diff
@@ -68,7 +68,7 @@ func (g *GSRelease) ReleaseComponents(version string) map[string]string {
 	return releaseComponents
 }
 
-func (g *GSRelease) Validate(version string) bool {
+func (g *Release) Validate(version string) bool {
```

Comment:
No blocker here, but should we change it to `r` for consistency?

Reply:
Sorted 🎉